Documentation/cluster-crd.md

```diff
@@ -73,14 +73,18 @@ Below are the settings available, both at the cluster and individual node level,
 
 ### Placement Configuration Settings
 
-Placement configuration for the cluster services. It includes the following keys: `api`, `mgr`, `mon`, `osd` and `all`. Each service will have its placement configuration generated by merging the generic configuration under `all` with the most specific one (which will override any attributes).
+Placement configuration for the cluster services. It includes the following keys: `api`, `mgr`, `osd` and `all`. Each service will have its placement configuration generated by merging the generic configuration under `all` with the most specific one (which will override any attributes).
```

Comment:
why remove the `mon` key from this paragraph?

Reply:
Right, added it again.